src/WebJobs.Script.WebHost/Management/GetTriggersResult.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management
+{
+    public class GetTriggersResult : TriggersOperationResult
```

Comment:
Not a fan of this `Get` prefix for this type.

Reply:
May be call it "TriggerInfo" or "TriggersResult" or "TriggersResponse" ?

Reply:
Changed to TriggersPayload